mtags/src/main/scala-3/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -173,12 +173,20 @@ object CaseKeywordCompletion:
       // Step 2: walk through known subclasses of sealed types.
       val sealedDescs = MetalsSealedDesc.sealedStrictDescendants(selectorSym)
       sealedDescs.foreach { sym =>
-        val autoImport = autoImportsGen.forSymbol(sym)
-        autoImport match
-          case Some(value) =>
-            visit(sym, sym.decodedName, value)
-          case scala.None =>
-            visit(sym, sym.showFullName, Nil)
+        // We don't want to import cases
+        if sym.isAllOf(Flags.EnumCase) || sym.isAllOf(Flags.EnumVal) then
+          visit(
+            sym,
+            s"${sym.owner.nameBackticked}.${sym.nameBackticked}",
+            autoImportsGen.forSymbol(sym.owner).getOrElse(Nil),
+          )
+        else
+          val autoImport = autoImportsGen.forSymbol(sym)
+          autoImport match
+            case Some(value) =>
+              visit(sym, sym.decodedName, value)
+            case scala.None =>
```

Comment:
It was probably there since the beginning, since I've copied some parts of code from 

https://github.com/scalameta/metals/blob/7c119f56746da63e5520b4332c90eeef43de7e40/mtags/src/main/scala-2/scala/meta/internal/pc/completions/MatchCaseCompletions.scala#L122-L123

Probably we can change it in both files

Reply:
Ach, that's fine. I was worried that metals added it automatically.